src/components/graph/graph.ts

```diff
@@ -30,11 +30,12 @@ import {
 } from './types'
 
 const logger = getLogger('d3-graph')
-if (window.localStorage.getItem('graphTransform') === undefined) {
+const temp1 = window.localStorage.getItem('graphTransform')
+if (temp1 === undefined || (temp1 !== null && temp1.match(/NaN/))) {
   window.localStorage.setItem('graphTransform', '0 0 1')
 }
 
-interface ILinkTuple { source: GraphNodeId, target: GraphNodeId }
```

Comment:
Leave it for now, no need to do more work. Since it's an interface, I leave the space around the braces. Same thing if it was a function, eg. `() => { console.log() }`